packages/babel-plugin-ember-test-metadata/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "babel-plugin-ember-test-metadata",
+  "version": "1.2.3",
+  "description": "A Babel plugin for Ember apps that adds a test's file path to test metadata",
+  "keywords": [],
+  "repository": "https://github.com/babel-plugin-ember-test-metadata/babel-plugin-ember-test-metadata",
+  "license": "MIT",
+  "author": {
+    "name": "Glenn Cueto",
+    "email": "gcueto@linkedin.com"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist/*.js"
+  ],
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "prebuild": "rm -rf dist",
+    "build": "babel src --out-dir dist --copy-files --ignore **/*.test.js",
+    "postbuild": "rm -rf dist/**/*.test.js",
+    "test": "BABEL_TEST_METADATA=true yarn test:jest",
+    "test:jest": "jest"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "**/src/*.js": [
+      "prettier --write"
+    ]
+  },
+  "prettier": {
+    "bracketSpacing": true,
+    "singleQuote": true,
+    "trailingComma": "es5"
+  },
```

Comment:
I think you'll want to copy this to be in the top level `package.json` too (this is why the top level `.eslintrc.js` needed single quote -> double quote changes)

Reply:
yep good call